packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -93,7 +94,8 @@ class InkResponse extends StatefulWidget {
     this.borderRadius: BorderRadius.zero,
     this.highlightColor,
     this.splashColor,
-  }) : super(key: key);
+    this.enableFeedback: true,
+  }) : assert(enableFeedback != null), super(key: key);
```

Comment:
Asserts are throwing left and right when I assert on containedInkWell and borderRadius. If the assert is really true we should do that (and the associated clean-up) in a follow-up PR.